webapp/src/main/java/com/box/l10n/mojito/okapi/filters/XMLEncoder.java

```diff
@@ -32,15 +34,20 @@ private boolean isAndroidStrings() {
             return false;
         }
     }
-    
+
     public String escape(String text) {
         boolean enclosedInDoubleQuotes = StringUtils.startsWith(text, "\"") && StringUtils.endsWith(text, "\"");
         if (enclosedInDoubleQuotes) {
             text = text.substring(1, text.length() - 1);
         }
-        
+
         String pattern = "(&lt;)(/?)(b|i|u)(&gt;)";
```

Comment:
because it will be called a bunch of time the pattern creation should be moved out of the function `Pattern.compile(regex)` and then do `pattern.matcher(str).replaceAll(repl) `